pkg/productinfo/alibaba/productinfo_alibaba.go

```diff
@@ -184,21 +189,40 @@ func (e *AlibabaInfoer) GetProducts(regionId string) ([]productinfo.VmInfo, erro
 	var vms []productinfo.VmInfo
 
 	request := ecs.CreateDescribeInstanceTypesRequest()
-	request.RegionId = regionId
+	request.RegionId = "eu-central-1"
 
 	vmSizes, err := e.client.DescribeInstanceTypes(request)
 	if err != nil {
 		return nil, err
 	}
+
+	dataFromJson, err := getJson()
+	if err != nil {
+		return nil, err
+	}
+
 	instanceTypes := vmSizes.InstanceTypes.InstanceType
 	for _, instanceType := range instanceTypes {
-		vms = append(vms, productinfo.VmInfo{
-			Type:    instanceType.InstanceTypeId,
-			Cpus:    float64(instanceType.CpuCoreCount),
-			Mem:     instanceType.MemorySize,
-			Gpus:    float64(instanceType.GPUAmount),
-			NtwPerf: fmt.Sprintf("%.1f Gbit/s", float64(instanceType.InstanceBandwidthRx)/1024000),
-		})
+		for key, prices := range dataFromJson.PricingInfo {
+			for _, price := range prices.Hours {
```

Comment:
we shouldn't iterate on this list and create a new VmInfo for each element. As far as I understand, `period` means that if you're reserving that instance for `period` `hours`/`months`/`years` then you'll pay that `price`. Because yet we only care about pay-as-you-go pricing, we should select the price with `"period": "1"` from the list and build the `VmInfo` based on that price. (I haven't gone though the json completely so maybe there's no such thing _now_ as "reserve for hours" and this list always contains one element only, but it'd still be safer if we could do this)